README.md

```diff
@@ -29,8 +29,6 @@ Install with:
 $ pip install chembl-downloader
 ```
 
-## Database Usage
```

Comment:
please revert this

Reply:
I no longer see this part in the README. Do you mean that I should remove the "ChEMBL Downloader Status" table?

Reply:
This diff shows you deleted this header. Please un-delete this header.

Reply:
Ah. Sorry about that. Fixed now.